src/user/test/all-get-user.controller.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { HttpStatus, ValidationPipe } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+
+import { NestApplication } from '@nestjs/core';
+import { UserController } from '../user.controller';
+import { UserInfoDto } from '../dto/user.dto';
+import { UserService } from '../user.service';
+import { mockUserService } from './mock/user.service.mock';
+import request from 'supertest';
+
```

Comment:
I will import it that way 

```
import { HttpStatus, ValidationPipe } from '@nestjs/common';
import { Test, TestingModule } from '@nestjs/testing';
import { NestApplication } from '@nestjs/core';
import request from 'supertest';

import { UserController } from '../user.controller';
import { UserService } from '../user.service';
import { UserInfoDto } from '../dto/user.dto';
import { mockUserService } from './mock/user.service.mock';
```

first libraries 
then space 
then import from the same folder next to each other